### DISCUSSION TITLE
Tracking item / package / car on Maps

### DISCUSSION THREAD

**Discussion by technologyconsulting** [notJust-dev/notjustdev#143] [Ideas]
Like viewing a taxi on Uber, a delivery driver from Amazon

**Comment by vladimir-vucetic** [comment]
For this I used a good package react-native-background-location but it is very expensive ($$$). It would be very cool if possible to see how to make native code for this and bridge to expo project. This could also provide learning resource how to make native plugins.

**Comment by tonven** [comment]
And if this is too big for 1 day delivery, maybe more like Airbnb map with points of interest on it. Where it will be possible to interact with points.

**Comment by iammanishk** [comment]
Is it possible to make live location tracking? Like WhatsApp live location share.

**Comment by khantanveer7** [comment]
https://www.youtube.com/live/ijBG81t0tAo?si=uRRBS8KmKYgwRfTj
